File: TransactionRegister/Controller/Reusable/TXTableView.m
//
//  TXTableView.m
//  TransactionRegister
//
//  Created by Eric Romrell on 10/25/16.
//  Copyright © 2016 Eric Romrell. All rights reserved.
//

#import "TXTableView.h"

@implementation TXTableView

-(instancetype)initWithCoder:(NSCoder *)aDecoder {
	self = [super initWithCoder:aDecoder];
	self.tableFooterView = [[UIView alloc] init];
	return self;
}

@end

File: TransactionRegister/Controller/Reusable/TXTableView.swift
//
//  TXTableView.swift
//  TransactionRegister
//
//  Created by Eric Romrell on 1/20/17.
//  Copyright © 2017 Eric Romrell. All rights reserved.
//

import UIKit

class TXTableView: UITableView {

	required init?(coder aDecoder: NSCoder) {
		super.init(coder: aDecoder)
		if self.tableFooterView == nil {
			self.tableFooterView = UIView()
		}
	}

}
